src/components/ReportActionItem/TransactionPreview/index.tsx

```diff
@@ -79,6 +79,14 @@ function TransactionPreview(props: TransactionPreviewProps) {
 
     const {originalTransaction, isBillSplit} = getOriginalTransactionIfBillIsSplit(transaction);
 
+    const IOUaction =
```

Comment:
```suggestion
    const iouAction =
```

Could be mistaken for a component name